src/github.com/getlantern/flashlight/ui/resources.go

```diff
@@ -1,7 +0,0 @@
-// This file will be overwritten by make genassets.
```

Comment:
Yeah, @myleshorton mentioned this too.  I don't understand the benefit to building with go directly instead of using make, and there's a clear downside to how we used to do it.  The build script doesn't push an updated resources.go, so anyone who builds the way that you suggest will be running with an outdated UI.  For example, the resources.go that's on GitHub hasn't been updated since September of last year.  I think it's best for us to run with the most recent code while we develop so that we can catch any errors that we might be introducing, so using make this way seems helpful.